src/renderer.js

```diff
@@ -43,7 +43,6 @@ setButton.addEventListener("click", () => {
 
   // Append the new <li> element to the <ul> element
   tasksList.appendChild(newTaskItem);
-  tomorrowTasksList.appendChild(newTaskItem);
```

Comment:
Removed this because it automatically appended all created tasks to the wrong columns